OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -70,6 +72,19 @@ void IRenderAboveWorld.RenderAboveWorld(Actor self, WorldRenderer wr)
 				TargetLineRenderable.DrawTargetMarker(wr, hc, hb);
 			}
 
+			var tc = Color.Lime;
+			var enabledPositions = targetablePositions.Where(Exts.IsTraitEnabled);
+			var positions = enabledPositions.SelectMany(tp => tp.TargetablePositions(self));
+			foreach (var p in positions)
+			{
+				var updown = new WVec(0, 128, 0);
+				var leftright = new WVec(128, 0, 0);
```

Comment:
I performed a test with 267 CYards on-screen, meaning 2403 targetable positions (+1 for my undeployed MCV).
There was no visible difference in ms/Tick between these vars and using `static readonly WVec`. In fact, `CombatDebugOverlay` appears to be astonishingly cheap on the simulation Tick side, only the rendering is expensive, and that doesn't change with those static readonly WVecs.
But I'll push the changes anyway, I want this merged quickly.